nsls2forge_utils/cli.py

```diff
@@ -123,3 +124,21 @@ def all_feedstocks():
         parser.exit(message='Please specify organization and sub-command...\n')
 
     args.func(args)
+
+
+def dashboard():
+    parser = argparse.ArgumentParser(
+        description='Create a dashboard of feedstocks belonging to nsls-ii-forge')
```

Comment:
I thought about that but decided against it due to the difference in naming `nsls-ii-forge` vs `nsls2forge`, unless some other organization is hosting our feedstocks on the same Azure and Anaconda profile.

I guess I didn't see the benefit in adding it since it would only change where the feedstocks are coming from. If you think I should still make the adjustment I will add it.

Reply:
OK, let's keep it as is for now.